### PR TITLE
Add Python2.7 ucs2+ucs4 variants

### LINKED ISSUE #1276: Python 2.7 Linux wheel without ucs4 support
Could a Python 2.7, Linux wheel be supported that doesn't have the --enable-unicode=ucs4 (-cp27mu-) requirement. Most Python Linux installs aren't usually compiled with this flag (e.g. Canopy Python package) in my experience. The OSX 2.7 wheel works well with just the  --enable-unicode=ucs2 flag, so I suspect it isn't critical.

### PR DESCRIPTION
**Pull request by lissyx**
Fixes #1276

### CODE REVIEW

File: tc-python-tests-prod.sh [hunk 1]
```diff
@@ -4,21 +4,23 @@ set -xe
...
+if [ -z "${pyver_full}" ]; then
     echo "No python version given, aborting."
     exit 1
 fi;
 
+pyver=$(echo "${pyver_full}" | cut -d':' -f1)
+
 # 2.7.x => 27
-pyver_pkg=$(echo "${pyver}" | cut -d'.' --output-delimiter="" -f1,2)
+pyver_pkg=$(echo "${pyver_full}" | cut -d':' -f1 | cut -d'.' -f1,2 | tr -d '.')
```
Comment (reuben) [inline tc-python-tests-prod.sh]: The first part of this, `echo "${pyver_full}" | cut -d':' -f1`, is just `$pyver`.

File: tc-python-tests.sh [hunk 2]
```diff
@@ -4,22 +4,24 @@ set -xe
...
+if [ -z "${pyver_full}" ]; then
     echo "No python version given, aborting."
     exit 1
 fi;
 
+pyver=$(echo "${pyver_full}" | cut -d':' -f1)
+
 # 2.7.x => 27
-pyver_pkg=$(echo "${pyver}" | cut -d'.' -f1,2 | tr -d '.')
+pyver_pkg=$(echo "${pyver_full}" | cut -d':' -f1 | cut -d'.' -f1,2 | tr -d '.')
```
Comment (reuben) [inline tc-python-tests.sh]: The first part of this, `echo "${pyver_full}" | cut -d':' -f1`, is just `$pyver`.

File: tc-tests-utils.sh [hunk 3]
```diff
@@ -482,8 +482,11 @@ do_deepspeech_python_build()
...
 
-  for pyver in ${SUPPORTED_PYTHON_VERSIONS}; do
-    pyenv install ${pyver}
+  for pyver_conf in ${SUPPORTED_PYTHON_VERSIONS}; do
+    pyver=$(echo "${pyver_conf}" | cut -d':' -f1)
+    pyconf=$(echo "${pyver_conf}" | cut -d':' -f2)
+
+    PYTHON_CONFIGURE_OPTS="--enable-unicode=${pyconf}" pyenv install ${pyver}
     pyenv virtualenv ${pyver} deepspeech
     source ${PYENV_ROOT}/versions/${pyver}/envs/deepspeech/bin/activate
```
Comment (reuben) [inline tc-tests-utils.sh]: Not related to this PR, but FWIW, this can be just `pyenv acticate ${pyver}/envs/deepspeech`.

Comment (reuben) [reply tc-tests-utils.sh]: activate, not acticate :)

Comment (lissyx) [reply tc-tests-utils.sh]: Yes, it should, but it always complained about something not being setup properly. I fighted that for hours, leading to nowhere. Poor error handling, poor debug handling, this was working this way so I kept it like that :)

**Pull request merged**